src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -99,13 +99,7 @@ def create_task_mappings(
         :param build_regex: Regex pattern to match build variant names against.
         :return: An instance of TaskMappings and version_id of the most recent version analyzed.
         """
-        log = LOGGER.bind(
-            project=evergreen_project,
-            module=module_name,
-            version_limit_stop_at_date=version_limit.stop_at_date,
-            version_limit_stop_at_version_id=version_limit.stop_at_version_id,
-        )
-        log.info("Starting to generate task mappings")
+        LOGGER.info("Starting to generate task mappings", version_limit=version_limit.__dict__)
```

Comment:
[nit] I think it would be better to define a `__repr__` method on `version_limit` instead of referencing `__dict__`.

Reply:
Oh whoops, forgot to remove the `__dict__` here